tests/test_venue_request_v2.py

```diff
@@ -514,10 +496,10 @@ def test_venue_recruitment_email_error(self, client, test_client, selenium, requ
         assert len(process_logs) == 1
         assert process_logs[0]['status'] == 'ok'
 
+        recruitment_status_invitation = '{}/-/Request{}/Recruitment_Status'.format(venue['support_group_id'], venue['request_form_note'].number)
         last_comment = client.get_notes(invitation=recruitment_status_invitation, sort='tmdate')[0]
 
-        assert '"OpenReviewException({\'name\': \'NotFoundError\', \'message\': \'Group Not Found: celeste@uni-a\'' in last_comment.content['error']
-        assert '"KeyError(\'name\')"' in last_comment.content['error']
+        assert '"InvalidGroup": [\n    "celeste@uni-a"\n  ]' in last_comment.content['error']
```

Comment:
I removed `{name}` in the invitation_email_content to avoid the token error. Somehow the error changed from Group Not Found to Invalid Group. Is this ok to change?

Reply:
yes, that's fine